packages/react-native-test-app-msal/android/build.gradle

```
@@ -74,6 +74,11 @@ repositories {
 }
 
 def rnxAuth = findNodeModulesPath("@rnx-kit/react-native-auth")
+def rnxDisableAuthProperty = "com.microsoft.reacttestapp.msal.disableRnxAuthModule"
+def skipAuthModuleRegistration =
+    rnxAuth != null &&
+    rootProject.hasProperty(rnxDisableAuthProperty) &&
+    rootProject.property(rnxDisableAuthProperty).toString().equalsIgnoreCase("true")
```

Actually, on closer inspection, this looks inverted. Shouldn't this be something like:
```suggestion
    rnxAuth == null ||
    (rootProject.hasProperty(rnxDisableAuthProperty) &&
        rootProject.property(rnxDisableAuthProperty).toString().equalsIgnoreCase("true"))
```

---

Yes, effectively doing this makes the logic further down a bit simpler! Thanks.